.gitignore

```diff
@@ -21,11 +21,15 @@ atlassian-ide-plugin.xml
 platform/solr/**/overlays
 distribution/docs/src/main/resources/content/**/.asciidoctor
 distribution/docs/src/main/resources/content/**/*.png
+*.class
```

Comment:
Compiled classes should be in the `target/` dir that's excluded below. I don't think we should need to add this 🤔